src/api/controller.js

```diff
@@ -493,6 +493,24 @@ export default class ApiController {
   async whoAmI(req, res) {
   }
 
+  async initialSuggestions(req, res) {
+    let User = this.bookshelf.model('User');
+
+    let users = await User.where('id', '!=', req.session.user).fetchAll({require: true, withRelated: ['posts']});
+    users = users.map(row => ({id: row.id, posts: row.relations.posts}));
+    users = users.sort((a,b) => b.posts.length - a.posts.length).map(row => row.id);
+
+    let q = User.forge()
+      .query(qb => {
+        qb
+          .whereIn('id', _.take(users, 20))
+      });
```

Comment:
can you do this on SQL side of the fence? join+count+group+order+limit or am I missing something?